sections/main-product.liquid

```diff
@@ -19,54 +19,146 @@
+                      <span class="thumbnail__badge" aria-hidden="true">
+                        {%- render 'icon-play' -%}
+                      </span>
+                    {%- endif -%}
+                    <button class="thumbnail {% if media.preview_image.aspect_ratio > 1 %}thumbnail--wide{% else %}thumbnail--narrow{% endif %}"
+                      aria-label="{%- if media.media_type == 'image' -%}{{ 'products.product.media.load_image' | t: index: media_index }}{%- elsif media.media_type == 'model' -%}{{ 'products.product.media.load_model' | t: index: media_index }}{%- elsif media.media_type == 'video' or media.media_type == 'external_video' -%}{{ 'products.product.media.load_video' | t: index: media_index }}{%- endif -%}"
+                      {% if media == product.selected_or_first_available_variant.featured_media or product.selected_or_first_available_variant.featured_media == null and forloop.index == 1 %} aria-current="true"{% endif %}
+                      aria-controls="GalleryViewer-{{ section.id }}"
+                      aria-describedby="Thumbnail-{{ section.id }}-{{ forloop.index }}"
+                    >
+                      <img id="Thumbnail-{{ section.id }}-{{ forloop.index }}"
+                        srcset="{% if media.preview_image.width >= 59 %}{{ media.preview_image | img_url: '59x' }} 59x,{% endif %}
+                                {% if media.preview_image.width >= 118 %}{{ media.preview_image | img_url: '118x' }} 118w,{% endif %}
+                                {% if media.preview_image.width >= 84 %}{{ media.preview_image | img_url: '84x' }} 84w,{% endif %}
+                                {% if media.preview_image.width >= 168 %}{{ media.preview_image | img_url: '168x' }} 168w,{% endif %}
+                                {% if media.preview_image.width >= 130 %}{{ media.preview_image | img_url: '130x' }} 130w,{% endif %}
+                                {% if media.preview_image.width >= 260 %}{{ media.preview_image | img_url: '260x' }} 260w{% endif %}"
+                        src="{{ media | img_url: '84x84' }}"
+                        sizes="(min-width: 1200px) calc((1200px - 19.5rem) / 12), (min-width: 750px) calc((100vw - 16.5rem) / 8), calc((100vw - 8rem) / 5)"
+                        alt="{{ media.alt | escape }}"
+                        height="200"
+                        width="200"
+                        loading="lazy"
+                      >
+                    </button>
+                  </li>
+                {%- endunless -%}
+              {%- endfor -%}
+            </ul>
+            <button type="button" class="slider-button slider-button--next{% if media_count <= 3 %} small-hide{% endif %}{% if media_count <= 4 %} medium-hide large-up-hide{% endif %}" name="next" aria-label="{{ 'general.slider.next_slide' | t }}" aria-controls="GalleryThumbnails-{{ section.id }}" data-step="3">{% render 'icon-caret' %}</button>
```

Comment:
I'm wondering if the next and previous buttons should be using `aria-hidden` ?
Because the thumbnail slider is a list, when I'm navigating through it, the only times I land on the previous or next arrow it's dimmed. 🤔  but maybe I'm missing use cases. [video](https://screenshot.click/21-12-mu3o3-0vk1h.mp4)

Reply:
> the only times I land on the previous or next arrow it's dimmed. 

Not necessarily true. Here's an example where I got to the `Next` arrow by using the Skip link and then tabbing back: https://user-images.githubusercontent.com/37168033/146236510-8076e15a-bbeb-48f7-a170-6aaf0bc4d930.mp4

Tabs will also cycle back, so if you tab through every element on the page you'll get back to the top, allowing you to focus on a `Prev` arrow:

https://user-images.githubusercontent.com/37168033/146236996-18a67cfd-a98d-478f-90e8-646f81c3bda4.mp4

In most cases, yes, you'll be able to navigate the thumbnails without using the arrows, but it's important that they are visible whenever you reach them via this non-linear paths.

Reply:
Nice. Very good 👍